apps/condo/domains/ticket/components/TicketId/TicketResidentFeatures.tsx

```diff
@@ -50,37 +51,52 @@ const getIsPaymentsInMobileAppAvailable = (receiptsInCurrentPeriod, receiptsInPr
     return false
 }
 
-const PaymentsAvailableIndicator: React.FC<PaymentsAvailableIndicatorProps> = ({ ticketOrganizationId }) => {
+const TWO_MONTHS_AGO = dayjs().startOf('day').subtract(60, 'days').toISOString()
```

Comment:
Why not to `dayjs().startOf('day').subtract(2, 'months').toISOString()` ?

Reply:
changed